apps/passport-client/src/useSyncE2EEStorage.tsx

```diff
@@ -181,15 +180,3 @@ export function useSyncE2EEStorage() {
     load();
   }, [load]);
 }
-
```

Comment:
useHasUploaded was unused, and depended on an old assumption that the uploadedId depended only on the PCDCollection, so would be wrong if someone tried to use it.  There is other unused code related to PCDCollection hashes, which maybe should be either removed or converted to hash more things, but it's non-dangerous for now.